unittests/CppInterOp/InterpreterTest.cpp

```diff
@@ -200,7 +203,13 @@ TEST(InterpreterTest, ExternalInterpreterTest) {
 #endif
 
   EXPECT_NE(ExtInterp, nullptr);
-  Cpp::UseExternalInterpreter(ExtInterp);
+
+#if !defined(NDEBUG) || !GTEST_HAS_DEATH_TEST
+  // death tests don't work right on windows
+  #ifndef _WIN32
```

Comment:
I thought this was covered by `GTEST_HAS_DEATH_TEST`. Did anything in particular fail?

Reply:
Yes, it doesn't hit the assertion:
`  1: Actual msg:
  1: [  DEATH   ] Unable to find target for this triple (no targets are registered)
  `

Reply:
Shouldn't we do `#if !defined(NDEBUG) && !GTEST_HAS_DEATH_TEST`?

Reply:
We do that, we do not need the `_WIN32` thing...

> Yes, it doesn't hit the assertion:
> 1: Actual msg: 1: [  DEATH   ] Unable to find target for this triple (no targets are registered)
 
 Seems a different problem...